### PR TITLE
KCE-68 Remove completely DDS from kpsr-build

### PR DESCRIPTION
**Pull request by joseluisbf-kpsr**

### CODE REVIEW

File: cmake/CodeFormat.cmake [hunk 1]
```diff
@@ -80,8 +80,7 @@ if(CMAKE_CODE_FORMAT)
         "CodeCoverage.cmake"
         "CppcheckTargets.cmake"
         "Findcppcheck.cmake"
-        "Findlog4cpp.cmake"
-        "FindRTIConnextDDS.cmake")
+        "Findlog4cpp.cmake")
```
Comment (pablogh-2000) [inline cmake/CodeFormat.cmake:83]: Do we need `Findlog4cpp`? Can you check what happens if we remove this dep?

File: cmake/MacroConnext.cmake [hunk 2]
```diff
@@ -12,17 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-##############################################################################
```
Comment (pablogh-2000) [inline cmake/MacroConnext.cmake:15]: Remove this file altogether.

Comment (joseluisbf-kpsr) [reply cmake/MacroConnext.cmake:15]: It builds and tests flawlessly: file removed.

**Comment by joseluisbf-kpsr** [comment]
Waiting for [KCE-68](https://github.com/klepsydra-technologies/kpsr-build/pull/20) merge of submodule `kpsr-build`.

**Comment by mharshe** [comment]
Please squash commits first and then I'll approve.

**Review by mharshe — APPROVED** [review]

**Pull request merged**